File: sdk/src/main/java/org/coralibre/android/sdk/internal/datatypes/IntervalOfCapturedData.java
package org.coralibre.android.sdk.internal.datatypes;

import java.util.ArrayList;
import java.util.List;

public class IntervalOfCapturedData {

    private final ENInterval interval;
    private final List<CapturedData> capturedData = new ArrayList<>();


    public IntervalOfCapturedData(ENInterval interval) {
        this.interval = interval;
    }

    public void add(final CapturedData capturedData) {
        this.capturedData.add(capturedData);
    }


    public ENInterval getInterval() {
        return interval;
    }

    public List<CapturedData> getCapturedData() {
        return capturedData;
    }
}

File: sdk/src/main/java/org/coralibre/android/sdk/internal/datatypes/IntervalOfCapturedData.kt
package org.coralibre.android.sdk.internal.datatypes

import java.util.ArrayList

class IntervalOfCapturedData(val interval: ENInterval) {
    private val capturedData: MutableList<CapturedData> = ArrayList()
    fun add(capturedData: CapturedData) {
        this.capturedData.add(capturedData)
    }

    fun getCapturedData(): List<CapturedData> {
        return capturedData
    }
}
